examples/material_gallery/lib/demo/weathers_demo.dart

```diff
@@ -0,0 +1,495 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' as ui;
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_sprites/flutter_sprites.dart';
+
+ImageMap _images;
```

Comment:
these are file-global, but you initialise them afresh in each call to `_WeathersDemoState.initState()`. They should probably be state in the `_WeathersDemoState` class.